src/hotspot/share/jeandle/jeandleRuntimeRoutine.cpp

```diff
@@ -91,3 +91,13 @@ JRT_END
 JRT_LEAF(address, JeandleRuntimeRoutine::get_exception_handler(JavaThread* current))
   return SharedRuntime::raw_exception_handler_for_return_address(current, current->exception_pc());
 JRT_END
+
+
+//=============================================================================
+//                      Array Allocation Routines
+//=============================================================================
+JRT_ENTRY(void, JeandleRuntimeRoutine::new_typeArray(int type, int length, JavaThread* current))
+  oop obj = oopFactory::new_typeArray(static_cast<BasicType>(type), length, current);
+  current->set_vm_result(obj);
+JRT_END
```

Comment:
This routine's implementation can refer to `OptoRuntime::new_array_C`.
I think it's better to use `Klass` pointer as argument like `OptoRuntime::new_array_C`. This approach will make it more convenient for us to obtain type information in the future.